nixos/modules/virtualisation/docker.nix

```diff
@@ -157,6 +273,8 @@ in
       systemd.services.docker = {
         wantedBy = optional cfg.enableOnBoot "multi-user.target";
         environment = proxy_env;
+        postStart = concatStrings (mapAttrsToList mkNetwork cfg.networks)
```

Comment:
shouldn't this be in a prestart for the thing running the containers?

Reply:
Well, the thing is, named volumes and networks are created not for the containers but for the system. That's why I put all of this on the docker module.

How would it look if I put all of this in `docker-containers.nix` anyway?

Reply:
For the record, I don't mind this being here, given that these are in essence "docker-global" entities, rather than being tied to any specific containers.

Reply:
hmm okay.